date/date_c.php

```diff
@@ -80,7 +80,7 @@ interface DateTimeInterface
      * @link https://secure.php.net/manual/en/datetime.diff.php
      * @param DateTimeInterface $targetObject <p>The date to compare to.</p>
      * @param bool $absolute <p>Should the interval be forced to be positive?</p>
-     * @return DateInterval
+     * @return DateInterval|false
```

Comment:
According to https://www.php.net/manual/en/datetime.diff.php and https://github.com/php/php-src/blob/master/ext/date/php_date.stub.php#L132 return type is just `DateInterval`

Reply:
That's strange - the official documentation says in the return value "or false on failure", but you're right, it is not mentioned anywhere else. I'll change it.